### DISCUSSION TITLE
default-production.pack ENOENT: no such file or directory,

### DISCUSSION THREAD

**Discussion by 782042369** [webpack/webpack#16009] [Q&A]
infrastructureLogging: { debug: true, colors: true },
[webpack.cache.PackFileCacheStrategy] No pack exists at /node_modules/.cache/default-production.pack: Error: ENOENT: no such file or directory, stat 'node_modules/.cache/default-production/index.pack'
How to configure it

**Comment by vankop** [comment]
thats ok, this log shows that there is no cache, after first build it will exist

**Reply by 782042369** [reply]
<img width="402" alt="image" src="https://user-images.githubusercontent.com/28727900/176936024-6020f623-b9ca-4a71-b0b3-769c5c8892b6.png">
Build multiple no such file。

**Reply by 782042369** [reply]
use cache builds are slow.
first build
<img width="451" alt="image" src="https://user-images.githubusercontent.com/28727900/176936618-b1c98abf-0b34-498f-b87e-a4f9922ecb99.png">
second build
<img width="463" alt="image" src="https://user-images.githubusercontent.com/28727900/176936694-8e267bd1-902c-4d32-8d40-cbe1461be763.png">

**Reply by alexander-akait** [reply]
x2 time, but yes, still slow, maybe you have plugins which don't support cache or loader which mark modules as noncachable, we need to look at your configuration to check it

**Reply by 782042369** [reply]
https://github.com/782042369/webpackconfig
this webpack config。some private code is not easy to post。

**Reply by 782042369** [reply]
no chage,sometimes build quickly.
<img width="459" alt="image" src="https://user-images.githubusercontent.com/28727900/176981314-1ef17990-3f85-478f-bf45-459d44c4a6e8.png">

**Reply by vankop** [reply]
try to use default `snapshot` configuration removing https://github.com/782042369/webpackconfig/blob/main/config/webpack.config.js#L184 
_________

default webpack is unnecessary here https://github.com/782042369/webpackconfig/blob/main/config/webpack.config.js#L211 (maybe even wrong)

**Reply by 782042369** [reply]
[webpack.cache.PackFileCacheStrategy] No pack exists at /Users/node_modules/.cache/webpack/default-production.pack: Error: ENOENT: no such file or directory, stat '/Users/node_modules/.cache/webpack/default-production/index.pack'
<t> [webpack.cache.PackFileCacheStrategy] restore cache container: 59.038147 ms
    [IdleFileCachePlugin] Initial cache was generated and cache will be persisted in 5s.
 After modification log。

**Reply by vankop** [reply]
logs looks correct one. first line shows that you don't have a cache pack

**Reply by 782042369** [reply]
<img width="731" alt="image" src="https://user-images.githubusercontent.com/28727900/177720891-02beb477-fa0e-4934-b567-a1fc4bfd7fde.png">
cache config

**Reply by vankop** [reply]
could you pls reduce to small reproducible repo?

**Comment by 782042369** [comment]
[bundle-stats.json.zip](https://github.com/webpack/webpack/files/9032180/bundle-stats.json.zip)
